db/migrations/mysql/000098_create_post_acknowledgements.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS PostAcknowledgements (
+    PostId varchar(26) NOT NULL,
+    UserId varchar(26) NOT NULL,
```

Comment:
nit: in a couple of places (e.g. parameters, model definiiton) I read the order `UserId, PostId` and in other cases, like here, I see `PostId, UserId`. I think this order makes sense -- should we be consistent everywhere?